dlint/linters/input_password_ask.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+from . import base
+
+
+class InputPasswordUseLinter(base.BaseLinter):
+    """This linter looks for use of the Python "input" function with a string
+    argument containing the word "password" in any format.
+    """
+    off_by_default = False
+
+    _code = 'DUO139'
+    _error_tmpl = 'DUO139 avoid using "input" to ask for password'
```

Comment:
I try to include the remediation in the message if it's short enough. How does this sound?

`DUO139 avoid using "input" to ask for password, prefer "getpass.getpass"`